packages/core/components/Modal/MoveFileManifest/MoveFileManifest.module.css

```diff
@@ -1,23 +1,43 @@
-.fileListContainer {
-    max-height: 200px;
+.fileTableContainer {
+    max-height: 300px;
     overflow-y: auto;
-    padding: 8px;
     border: 1px solid #ddd;
-    border-radius: 4px;
+    margin-bottom: 1em;
 }
 
-.fileList {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
+.fileTable {
+    width: 100%;
+    border-collapse: collapse;
 }
 
-.fileItem {
-    margin-bottom: 4px;
+.fileTable th,
+.fileTable td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
```

Comment:
throughout this file: ideally we should aim to only use the closest matching global.css variable colors for borders, backgrounds, etc

Reply:
resolved in [588447d](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/588447da65efe53f1b4eebc5459cc97f4b185646)